code/game/machinery/computer/tram_controls.dm

```diff
@@ -9,6 +9,8 @@
 
 	light_color = LIGHT_COLOR_GREEN
 
+	var/datum/weakref/tram_ref
```

Comment:
Document this because it's used outside of just tram_controls (in the component)

Last request

Reply:
ah excuse me